test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs

```diff
@@ -162,6 +226,17 @@ private static int GetCultureInfoDepth(CultureInfo culture)
             return result;
         }
 
+
+        private TestSink _sink = new TestSink();
```

Comment:
These need to go at the top of the class. Also

```C#
private readonly TestSink _sink = new TestSink();
private readonly ILogger Logger { get; } = ew TestLoggerFactory(_sink, true).CreateLogger<ResourceManagerStringLocalizer>();
```

^ does the second line work?